tools/rosbag/test/test_roundtrip.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+import rosbag
+import rospy
+
+class TestRoundTrip(unittest.TestCase):
+    def _write_simple_bag(self):
+        from std_msgs.msg import Int32, String
+
+        with rosbag.Bag('/tmp/test.bag', 'w') as bag:
```

Comment:
Does this need to also be unique to each `test_...` function? Or are those tests guaranteed to run sequentially?